config/sync/field.storage.media.field_media_embeddable_oembed.yml

```diff
@@ -0,0 +1,25 @@
+uuid: 36c142ba-b550-414d-824c-c5ed220eb0f9
+langcode: en
+status: true
+dependencies:
+  module:
+    - field_permissions
+    - media
+third_party_settings:
+  field_permissions:
+    permission_type: public
```

Comment:
Remove the field permissions if they are public. I know it does it automatically but its cleaner if we can.

Reply:
fixed!